airbyte-integrations/connectors/source-mailgun/source_mailgun/spec.json

```diff
@@ -9,7 +9,12 @@
     "properties": {
       "private_key": {
         "type": "string",
-        "description": "Private API key for authenticate with every request"
+        "description": "Private API key for authenticate with every request",
+        "airbyte_secret": true
+      },
+      "start_timestamp": {
```

Comment:
that is not very convenient as you can imagine, better use str and format date-time, you can check places in other connectors with name `Replication Start Date`

Reply:
I being thinking to move such things in CDK to avoid situation like this